modules/swagger-codegen/src/main/resources/TypeScript-node/api.mustache

```diff
@@ -50,40 +49,36 @@ interface Authentication {
 class HttpBasicAuth implements Authentication {
```

Comment:
If we don't actually use these other auth types, maybe easier to just remove them?